lib/reversetunnel/site.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Functions that are used in both local and remote sites.
```

Comment:
I would remove this, as comments on the package clause should only be in one file for package level godoc. We already have nice package-level documentation for this package:

```
➜ go doc lib/reversetunnel
package reversetunnel // import "github.com/gravitational/teleport/lib/reversetunnel"

Package reversetunnel sets up persistent reverse tunnel between remote site and
teleport proxy, when site agents dial to teleport proxy's socket and teleport
proxy can connect to any server through this tunnel.
```